Makefile

```
@@ -208,3 +214,22 @@ test-e2e: ## Runs e2e tests.
 .PHONY: test-e2e
 test-e2e: deploy
 	go test -timeout 20m -count 1 -v -tags e2e -run "$(TEST)" ./test/e2e
+
+# OLM
+.PHONY: bundle
+bundle: manifests
+	operator-sdk generate kustomize manifests -q
+	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(SEMVER)
+	operator-sdk bundle validate ./bundle
+
+.PHONY: bundle-build
+bundle-build:
+	$(CONTAINER_ENGINE) build -t $(IMAGE)-bundle:$(VERSION) -f bundle.Dockerfile
```

Might be safer to use quotes here and in the other recipes in this file:
```suggestion
	"$(CONTAINER_ENGINE)" build -t "$(IMAGE)-bundle:$(VERSION)" -f bundle.Dockerfile
```

---

Updated all similar recipes with quotes